integrations/amazonKeys/cli/init.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+readonly IFS=$'\n\t'
+readonly ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SCRIPT_NAME="$( basename "$0" )"
+readonly ARGS=("$@")
+
+export RESOURCE_NAME=""
+export SCOPE=""
+
+print_help() {
+  echo "
+  Usage:
+    $SCRIPT_NAME <resource_name> [scope]
+  "
+}
+
+parse_args() {
+  if [ "$#" -gt 0 ]; then
+    key="$1"
+    case "$key" in
+      --help)
+        print_help
+        exit 0
+        ;;
+      *)
+        RESOURCE_NAME=$1
+        SCOPE=$2
+        if [ "$SCOPE" == "" ]; then
+          SCOPE="configure"
+        fi
+        ;;
+    esac
+  else
+    print_help
+    exit 1
+  fi
+}
+
+init() {
+  echo "Setting up $SCOPE for resource $RESOURCE_NAME."
+}
+
+main() {
+  parse_args "${ARGS[@]}"
```

Comment:
Passing an entire array is a bit more involved now. Is this okay?

Reply:
this is fine